Patches/Alpha Animals/ThingDefs_Races/AlphaAnimals_CE_Patch_Race_Mammoth_Worm.xml

```diff
@@ -18,28 +18,30 @@
 			</li>
 
 			<li Class="PatchOperationReplace">
-		<xpath>/Defs/PawnKindDef[defName="AA_MammothWorm"]/combatPower</xpath>
-		<value>
-			<combatPower>230</combatPower>
-		</value>
+				<xpath>/Defs/PawnKindDef[defName="AA_MammothWorm"]/combatPower</xpath>
+				<value>
+					<combatPower>250</combatPower>
+				</value>
 			</li>
 			
 			<li Class="PatchOperationReplace">
 				<xpath>/Defs/ThingDef[defName="AA_MammothWorm"]/statBases/ArmorRating_Blunt</xpath>
 				<value>
-					<ArmorRating_Blunt>26</ArmorRating_Blunt>
```

Comment:
Why reduce the armor? Their speed really isn't that impressive, even with the change.